Update operational considerations (#81)
IETF-OPSAWG-WG/draft-opsarea-rfc5706bis: draft-opsarea-rfc5706bis.md

   This document obsoletes {{?RFC5706}} and fully updates its content
   with new operational and management techniques and mechanisms. It also
   introduces a requirement for an "Operational Considerations"

>    section in Internet-Drafts, before they are progressed

Reviewer: ```suggestion
   section in all Proposed Standard Track RFCs.
```

>    for publication as RFCs. Also, the document removes outdated

Reviewer: ```suggestion
   Also, the document removes outdated
```

   deployed protocols may have established de facto operations and
   management practices that are already well understood. However, the guidance
   in this document does not mandate providing a comprehensive inventory of
   all operational considerations. Instead, the guidance is to
   focus on key aspects that will ease deployability and are essential for operating and maintaining
   the technology.

   Suitable management approaches may vary for different areas, working
   groups, and protocols in the IETF. This document does not prescribe

>    a fixed solution or format in dealing with operational

Reviewer: ```suggestion
   a fixed solution or format in dealing with operational and management
```

   * Move the "Operational Considerations" Appendix A to a Checklist maintained in GitHub

>    * Add a requirement for an "Operational Considerations" section in all Internet-Drafts, along with specific guidance on its content.

Reviewer: ```suggestion
   * Add a requirement for an "Operational Considerations" section in all PS RFCs, along with specific guidance on its content.
```

      * Update the "Available Management Technologies" landscape

>    * Add an "Operational Tooling Considerations" section

Reviewer: ```suggestion
   * Add an "Operational and Management Tooling Considerations" section
```

#  Documentation Requirements for IETF Specifications {#sec-doc-req-ietf-spec}

> ##  Operations Considerations Section {#sec-oper-manag-considerations}

Reviewer: ```suggestion
##  Operational Considerations Section {#sec-oper-manag-considerations}
```

   After evaluating the operational and manageability aspects of a
   protocol, a Protocol Extension, or an architecture, the resulting practices and
   requirements should be documented

>    in an "Operations Considerations" section within a

Reviewer: ```suggestion
   in an "Operational Considerations" section within a
```

   Protocol Designers should consider both data-oriented and task-
   oriented authority levels and policy.

> # Operational Tooling Considerations {#sec-oper-mgmt-tooling}

Reviewer: ```suggestion
# Operational and Management Tooling Considerations {#sec-oper-mgmt-tooling}
```
